packages/serverless-contracts/src/contracts/apiGateway/types/lambdaHandler.ts

```diff
@@ -14,7 +18,7 @@ import {
   OutputType,
 } from './common';
 import { InputSchemaType } from './input';
-import type { DefinedProperties } from './utils';
+import { DefinedProperties } from './utils';
```

Comment:
Why is this necessary?

Reply:
Don't know why i had to add it in the first place, but it works without it now